pom.xml

```diff
@@ -198,6 +200,35 @@
                     <artifactId>central-publishing-maven-plugin</artifactId>
                     <version>0.8.0</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
yes, but no. Why is there no (SSOT) parent doing this like in maven?

Now we having this problem 2 times, replacing it with the next tool.

- https://github.com/apache/maven-parent/blob/f4ab7c2eb04865071186c57a263243c72b6e8d52/pom.xml#L1212